cmd/download_test.go

```diff
@@ -87,22 +87,22 @@ func TestDownload(t *testing.T) {
 	}()
 
 	testCases := []struct {
-		requestor   string
+		requester   bool
 		expectedDir string
 		flags       map[string]string
 	}{
 		{
-			requestor:   requestorSelf,
+			requester:   true,
 			expectedDir: "",
 			flags:       map[string]string{"exercise": "bogus-exercise"},
 		},
 		{
-			requestor:   requestorSelf,
+			requester:   true,
 			expectedDir: "",
 			flags:       map[string]string{"uuid": "bogus-id"},
 		},
 		{
-			requestor:   requestorOther,
```

Comment:
These constants are still defined below but no longer used

Reply:
Nice, thank you for catching that.